pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTExpressionStatement.java

```diff
@@ -42,4 +45,6 @@ public int getBeginColumn() {
 
         return super.getBeginColumn() - beginColumnDiff;
     }
+     */
+    // TODO(b/239648780)
```

Comment:
Just curious: can you explain your TODO organization? It seems you're referencing here something with "b/239648780" :)

Reply:
Ah, fair question. :) We have an internal policy that every code TODO must have a link to a tracker issue or a responsible person.  For these necessarily temporary placeholders, they all link to b/239648780 = "Convert PMD Apex module from Jorje to Summit AST". It would probably be better to use the GitHub issue -- will use that in future TODOs.